cmd/bosun/web/static/js/config.ts

```diff
@@ -54,6 +57,8 @@ bosunControllers.controller('ConfigCtrl', ['$scope', '$http', '$location', '$rou
 	$scope.template_group = search.template_group || '';
 	$scope.items = parseItems();
 	$scope.tab = search.tab || 'results';
+	$scope.aceTheme = 'chrome';
+	$scope.aceMode = 'bosun';
```

Comment:
They get used again in the toggle function. Plus might want to make theme changeable at some point.